File: FizzBuzz/fizzbuzz.js
for(i=0;++i<101;)console.log(i%15<1&&"FizzBuzz"||i%3<1&&"Fizz"||i%5<1&&"Buzz"||i)
for(i=0;++i<101;){console.log("Fizz".repeat(+i%3<1)+"Buzz".repeat(+i%5<1)||i)}

File: FizzBuzz/fizzbuzz.py
for i in range(1, 101):
    a = ''
    if not i % 3:
        a += 'Fizz'
    if i % 5:
        a += 'Buzz'
    if a == '':
        a = str(i)
    print(a)

for i in range(1,101):print("Fizz"*(i%3<1)+(i%5<1)*"Buzz"or i)
